includes/class-sensei-teacher.php

```diff
@@ -440,7 +440,14 @@ public static function update_course_modules_author( $course_id ,$new_teacher_id
                         // add the new term, the false at the end says to replace all terms on this module
                         // with the new term.
                         wp_set_object_terms( $lesson->ID, $term_id , 'module', false );
-                        update_post_meta( $lesson->ID, '_order_module_' . intval( $term_id ), 0 );
+                        $order_module = 0;
+                        if ( $old_order_module = get_post_meta( $lesson->ID, '_order_module_' . intval( $term->term_id ), true ) ) {
```

Comment:
can retrieving the post_meta and the if check be on separate lines?

Reply:
They _can_. 😉